sites/website/src/prepare-docs.js

```diff
@@ -48,77 +61,53 @@ function identifyPackage(path) {
 
 async function safeCopy(source, dest) {
     if (fs.existsSync(dest)) {
-        await fse.copyFile(source, dest);
+        await fs.copyFile(source, dest);
     } else {
-        await fse.mkdir(path.dirname(dest), { recursive: true });
-        await fse.copyFile(source, dest);
+        await fs.mkdir(path.dirname(dest), { recursive: true });
+        await fs.copyFile(source, dest);
     }
 }
 
 async function safeWrite(dest, content) {
     if (fs.existsSync(dest)) {
-        await fse.writeFile(dest, content);
+        await fs.writeFile(dest, content);
     } else {
-        await fse.mkdir(path.dirname(dest), { recursive: true });
-        await fse.writeFile(dest, content);
+        await fs.mkdir(path.dirname(dest), { recursive: true });
+        await fs.writeFile(dest, content);
     }
 }
 
 async function moveMarkdownFiles(src, docsFolderDest) {
     const files = findFiles(src, ".md");
     for (const source of files) {
         const filename = path.basename(source);
-        const root = "./docs";
-        const dest = path.join(root, docsFolderDest, filename);
+        const dest = path.join(__dirname, "../docs", docsFolderDest, filename);
 
         await safeCopy(source, dest);
     }
 }
 
 async function copyArticleMarkdown() {
     await moveMarkdownFiles(
-        "../../packages/web-components/fast-foundation/docs/integrations",
+        path.resolve(fastFoundation, "docs/integrations"),
```

Comment:
Yeah...seems like I forgot about `path.resolve` when originally whipping some of this up.... 🤣